test/e2e/operator_test.go

```diff
@@ -2596,75 +2604,84 @@ func TestContainerLoggingMinLength(t *testing.T) {
 }
 
 func TestIngressControllerCustomEndpoints(t *testing.T) {
+	if infraConfig.Status.ControlPlaneTopology == configv1.ExternalTopologyMode {
+		t.Skipf("skipping TestIngressControllerCustomEndpoints test due to controlplanetopology: %v", infraConfig.Status.ControlPlaneTopology)
+	}
```

Comment:
What concerns me about this skip is that we didn't skip on External control plane typologies before. And now, in an attempt to deflake we are skipping it on a subset of "architectures". Do you think we can skip the CCM wait check if the topology is external?